internal/config/config.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import "github.com/spf13/viper"
+
+type Config struct {
+}
+
+type PGConfig struct {
+	Host     string
+	Port     int
+	User     string
+	Password string
+	DBName   string
+}
+
+func New() *Config {
+	return &Config{}
```

Comment:
```suggestion
	return &Config{}
```
Remove this. Why do you need constructor of struct and do not use?